To: vim_dev@googlegroups.com
Subject: Patch 7.3.211
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.211 (after 7.3.210)
Problem:    Compiler warning.
Solution:   Add type cast.
Files:      src/if_cscope.c


*** ../mercurial/vim73/src/if_cscope.c	2011-06-12 20:42:17.000000000 +0200
--- src/if_cscope.c	2011-06-12 21:22:53.000000000 +0200
***************
*** 2518,2524 ****
  	/* Check for csdir to be non empty to avoid empty path concatenated to
  	 * cscope output. TODO: avoid the unnecessary alloc/free of fullname. */
  	vim_free(fullname);
! 	fullname = concat_fnames(csdir, (char_u *)name, TRUE);
      }
      else
  	(void)sprintf(fullname, "%s", name);
--- 2518,2524 ----
  	/* Check for csdir to be non empty to avoid empty path concatenated to
  	 * cscope output. TODO: avoid the unnecessary alloc/free of fullname. */
  	vim_free(fullname);
! 	fullname = (char *)concat_fnames(csdir, (char_u *)name, TRUE);
      }
      else
  	(void)sprintf(fullname, "%s", name);
*** ../vim-7.3.210/src/version.c	2011-06-12 20:42:17.000000000 +0200
--- src/version.c	2011-06-12 21:24:03.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     211,
  /**/

-- 
Did you ever see a "Hit any key to continue" message in a music piece?

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
